src/utils/daos.c

```diff
@@ -1071,8 +1071,63 @@ print_oclass_names_list(FILE *stream)
 	free(str);
 }
 
+#define FIRST_LEVEL_HELP() \
```

Comment:
(style)  macros should not use a trailing semicolon

Reply:
should have been resolved with next commit 7c41b2cd051243cf4560a225393628316c0a9901